Automattic/woocommerce-payments
# Add: Test Mode badge to Classic Checkout, Pay for Order, and Add Payment Method

Fixes #9487

#### Changes proposed in this Pull Request

* Add a "Test Mode" badge to Classic Checkout, Pay for Order, and Add Payment Method page that will match the badge already present in blocks checkout added by https://github.com/Automattic/woocommerce-payments/pull/9483
  * The styles are implemented using a grid layout to maintain consistency when the label size varies.

#### Testing Instructions - Prerequisites

* Enable test mode.
* Enable all payment methods.
* Enable the following currencies: USD, EUR
* Go to checkout

#### Testing Instructions - Checkout

* Add a product to the cart over 50 USD and go to classic checkout
* Verify that the test mode badge is displayed and that the layout is consistent whether the options are collapsed or expanded and in different resolutions.
  * Desktop
![image](https://github.com/user-attachments/assets/e94efb5d-1687-43cb-a92f-d9f4b85be550)
  * Mobile 
![image](https://github.com/user-attachments/assets/a1c49fef-9f8e-470e-8819-b5b9de8cfb6f)
* Change the currency to EUR, verify that the payment methods are properly hidden
  * ![image](https://github.com/user-attachments/assets/ecae03e4-735b-4442-8b89-0e764c52fa16)
* Change the address country between Belgium, Netherlands, Poland and verify that payment methods dynamically show/hide
  * ![image](https://github.com/user-attachments/assets/e67903d2-8d5b-408d-adc9-99a5abc98bd0) 
* Temporarily disable test mode and validate that the layout is consistent, but the badge is no longer displayed
  * Desktop
![image](https://github.com/user-attachments/assets/5bfaf6f7-f234-4afa-bebc-02b23e231675)
  * Mobile
![image](https://github.com/user-attachments/assets/89582afd-6d8d-4e79-a994-c370cc36413e)
* Re-enable test mode

#### Testing Instructions - Add Payment Method

* Go to My Account > Payment Methods > Add Payment Method
* Validate that the test mode badge is displayed
  * ![image](https://github.com/user-attachments/assets/288f67ec-9750-499a-a99b-12a10f5f3625)
* Temporarily disable test mode and validate that the layout is preserved Re-enable test mode afterwards.

#### Testing Instructions - Pay for order
 
* In WP Admin, create an order and open the customer payment page as a shopper
* Validate that the test mode badge is displayed
  * ![image](https://github.com/user-attachments/assets/cd668c6d-969d-4512-afeb-9a861c042cce)
 
#### Testing Instructions - Regression
* In WP Admin, open a paid order.
* Validate that the payment method string ONLY includes the name, and no additional characters/HTML is rendered.
  * ![image](https://github.com/user-attachments/assets/0da82acb-d929-4561-9335-4336c580f4d7)


-------------------


**Post merge**
